app/src/main/java/com/eventyay/organizer/utils/DateUtils.java

```diff
@@ -48,10 +48,10 @@ private static String formatDate(@NonNull String format, @NonNull ZonedDateTime
 
     @NonNull
     private static ZoneId getZoneId() {
-        if (showLocal || ContextManager.getSelectedEvent() == null)
+//        if (showLocal || ContextManager.getSelectedEvent() == null)
             return ZoneId.systemDefault();
-        else
-            return ZoneId.of(ContextManager.getSelectedEvent().getTimezone());
```

Comment:
@GOVINDDIXIT What's the need of removing this? You can't just return the default system `ZoneId` for any custom timezone. Every timezone has it's own `ZoneId` which should be returned by `getZoneId()` method.This is the reason for failing tests in `DateTimeUtilsTest`.

Reply:
Yes, the reason for failing DateUtilsTest is above mentioned changes, I only did these changes to show that the CreateSessionViewModelTests are failing due to ZoneId becoming null.